express-forms/src/index.js

```diff
@@ -1,19 +1,35 @@
 const express = require('express');
+const path = require('path');
+const bodyParser = require('body-parser');
 
 const app = express();
 
-app.get('/', (req, res) => {
-  return res.redirect('/form-with-get');
+app.use(bodyParser.urlencoded({ extended: true }));
+const publicPath = path.join(__dirname, './views');
+app.set('view engine', 'ejs');
+app.use(express.static(publicPath));
+
+
+app.get('/', (req, res) => res.redirect('/form-with-get'));
+
+app.get('/form-with-get', (req, res) => res.render('form-with-get'));
+
+app.get('/form-with-post', (req, res) => {
+  res.render('form-with-post');
 });
 
-app.get('/form-with-get', (req, res) => {
-  return res.render('form-with-get');
+app.get('/submit-form-with-get', (req, res) => {
+  const { first, last } = req.query;
+
+  res.send({ first, last });
 });
 
-app.get('/form-with-post', (req, res) => {
-  return res.render('form-with-post');
+app.post('/submit-form-with-post', (req, res) => {
+  console.log(req.body);
+  res.send(JSON.stringify(req.body));
```

Comment:
I feel like there is inconsistent use of `res.send`.
In route `GET /form-with-get`, you are passing `object` as response.
And here you are passing `string` as response.
Could you check with other routes also?